packages/present/README.md

```diff
@@ -0,0 +1,97 @@
+# Selective Disclosure JWT (SD-JWT) Draft 06 & Selective Disclosure JWT VC 01
```

Comment:
We can just link in the package.json to `../../README.md` right? 

I would like to have individual README's in the future though.